Ruddarr/Views/Shared/InformationList.swift

```diff
@@ -67,7 +67,7 @@ struct InformationItem: Hashable {
     var link: (any Hashable)?
 
     static func == (lhs: InformationItem, rhs: InformationItem) -> Bool {
-        lhs.label == rhs.label && lhs.value == rhs.value
+        lhs.label == rhs.label && lhs.value == rhs.value && equals(lhs, rhs)
```

Comment:
@tillkruss this was where we broke `Hashable` contract.

"Two instances that are equal must feed the same values to Hasher in hash(into:), in the same order."

An `InformationItem` without a link was equal to one with a link but had a different hash.

Reply:
Can this just be `lhs.link?.hashValue == rhs.link?.hashValue`?